update_external_sources.bat

```diff
@@ -14,7 +14,7 @@ git submodule update --init --recursive
    echo.
    echo Building %JSONCPP_DIR%
    cd  "%JSONCPP_DIR%"
-   python amalgamate.py
+   python3 amalgamate.py
```

Comment:
Windows doesn't have a `python3`.  There is a `py.exe` that takes `-2` or `-3` to select a version of Python.

CI just uses `python` (which has always been Python 3).

The approved generic call would be:
```
py -3 amalgamate.py
```
but it's feasible to just leave this `python`.

Reply:
My windows machine has a python3...?
py or python will NOT work on macOS. Hence the problem.